### DISCUSSION TITLE
Has IGR stopped working in recent versions?

### DISCUSSION THREAD

**Discussion by zilch3210** [emukidid/swiss-gc#891] [Q&A]
I'm running the latest version of Swiss with no issues booting. SD card is Fat32 and in the root of my card I have an ipl.dol and igr.dol file. My igr.dol file is just a copy of the ipl file.

Within the swiss config I've got "IGRType=igr.dol". The configuration parameter is set to the sd2sp2 card slot which seems right. I feel like I've checked all the right boxes but upon reset I'm sent to the original gamecube loading screen, not swiss. User error or a bug?

<img width="443" alt="Screenshot 2024-07-27 191524" src="https://github.com/user-attachments/assets/ff113616-349c-4ebe-af3a-3646dac9b3d4">

**Comment by twostepjake** [comment]
Having EXACT same issue. Would also like to know if this is a bug or we're forgetting something or not doing something right!

**Reply by altambai** [reply]
The same thing happens to me with the new version of Swiss, it doesn't exit the game. That's the first priority the system should have, exit the game and return to the Swiss menu.

traslate
tambien me  pasa  lo mismo la nueva version del swiss, no sale del juego eso es la primera prioridad q debe tener el sistema, salir del juego y volver al menu swiss

**Comment by altambai** [comment]
with this the problem is solved
Under the Swiss file, there's a folder called AppLoader. In this folder, there's a file called AppLoader.img. This must be installed at the root of your SD card. Also, install swiss/patches.
Remember that the Swiss folder on your SD card is hidden in the Swiss settings. You must enable the option to select hidden files and then check again to see if the folder has been unlocked to copy the AppLoader file to the Swiss\patches folder.

**Reply by Extrems** [reply]
Simply copy the contents of the `Apploader` folder into the root directory of your SD card and choose to overwrite. That's all you have to do.
